CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 - Added support for `to_boolean` function.
 - Added documentation pages for `Index` and its APIs.
+- Added support for `DataFrame.assign`.
```

Comment:
Can we add this to the Snowpark pandas APIs new features section? I also this the above item on Index should be Snowpark pandas?

Reply:
Moved both!